streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClientTest.java

```diff
@@ -0,0 +1,39 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements. See the NOTICE
+  * file distributed with this work for additional information regarding copyright ownership. The ASF licenses this file
+  * to You under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+  * License. You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  * specific language governing permissions and limitations under the License.
+  */
+
+package org.apache.kafka.streams.processor.internals;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.SaslConfigs;
+import org.apache.kafka.streams.StreamsConfig;
+import org.junit.Test;
+
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+public class StreamsKafkaClientTest {
+
+    @Test
+    public void testConfigFromStreamsConfig() {
+        Properties props = new Properties();
```

Comment:
I know this is a bit picky, but we (mostly @mjsax and i) are trying to encourage everyone to use `final` where possible